x-pack/plugins/security_solution/public/entity_analytics/components/entity_store/hooks/use_entity_store.ts

```diff
@@ -69,12 +70,17 @@ export const useEnableEntityStoreMutation = (options?: UseMutationOptions<{}>) =
 
 export const INIT_ENTITY_ENGINE_STATUS_KEY = ['POST', 'INIT_ENTITY_ENGINE'];
 export const useInitEntityEngineMutation = (options?: UseMutationOptions<{}>) => {
+  const isServiceEntityStoreEnabled = useIsExperimentalFeatureEnabled('serviceEntityStoreEnabled');
   const queryClient = useQueryClient();
 
   const { initEntityEngine } = useEntityStoreRoutes();
   return useMutation<InitEntityEngineResponse[]>(
-    () => Promise.all([initEntityEngine('user'), initEntityEngine('host')]),
-
+    () =>
+      Promise.all(
```

Comment:
There is [this ticket](https://github.com/elastic/security-team/issues/11309) addressing this. We don't need to do any changes in this PR.

This hook is not currently used AFAIK?